cloud/docker/docker.py

```diff
@@ -713,6 +721,8 @@ def __init__(self, module):
             self.links = self.get_links(self.module.params.get('links'))
 
         self.env = self.module.params.get('env', None)
+        self.env_file = self.module.params.get('env_file', None)
```

Comment:
I (force) pushed another commit which keeps the variables local to keep the code readable, but doesn't expose them in the module.